www/md_base/src/app/common/directives/inspectdata/inspectdata.tpl.jade

```diff
@@ -0,0 +1,13 @@
+div.inspect-data
+  div.row(layout="row", ng-repeat="item in inspectdata.items")
+    div.key(flex="33") {{ item.key }}
+    div.value(flex="66")
+      span(ng-if="item.value.type == 'string'") {{ item.value.value }}
+      a(ng-if="item.value.type == 'link'", ng-href="{{ item.value.value }}") {{ item.value.value }}
+      div.object-value(
+        ng-if="item.value.type == 'object'",
+        ng-init="collapsed = 1",
+        ng-click="collapsed = !collapsed",
```

Comment:
I think html complexity is no argument for directives. I indeed really would like you try with the sticker